pkg/agent/nplagent/lib/utils.go

```diff
@@ -0,0 +1,87 @@
+// +build !windows
```

Comment:
How about move this file to pkg/agent/util/parse_port.go

Reply:
Sorry, this is my mistake. Originally I was thinking about pkg/agent/nplagent/util/port_parser.go (for the package path to be consistent with other util code in the repo); but then I found maybe we can move/remote the other two funcs in this package/file, and started to wonder do we still need to keep the package just for a single func. But I forgot to update the comments.
I actually suggest to either remove the file and move the func to the caller, or move it to pkg/agent/nplagent/util/port_parser.go. What you think?

Reply:
That works. I'll move it to pkg/agent/nplagent/util/port_parser.go